### ISSUE TITLE
[BUG] Core dumped when use deviation calculate with multi models of float precision

### ISSUE THREAD

**Issue by jiongwalai** [deepmodeling/deepmd-kit#1068]
Hi, Developers:

**Summary**

When I use deviation calculate with multi models(float precision), terminal occur **segmentation fault (core dumped)** before MD. 
 However it works in single model in float precision or multi models in double precision.

**Deepmd-kit version, installation way, input file, running commands, error log, etc.**

**version:** v2.0.0
**installation way:** conda environment:
conda install deepmd-kit=*=*gpu libdeepmd=*=*gpu lammps-dp -c https://conda.deepmodeling.org
**input file:**
```
variable        NSTEPS1         equal 100000
variable        NSTEPS2         equal 5000
variable        NSTEPS3         equal 1
variable        THERMO_FREQ     equal 10
variable        DUMP_FREQ       equal 10
variable        TEMP1           equal 1650
variable        TEMP2           equal 300
variable        SEED            equal 12345
variable        TAU_T           equal 0.1

units           metal
boundary        p p p
atom_style      atomic

neighbor        1.0 bin

box          tilt large
read_data conf.lmp
change_box   all triclinic
mass            1 121.760000
pair_style      deepmd ../graph.002.pb ../graph.000.pb ../graph.001.pb ../graph.003.pb out_freq 10 out_file model_devi.out
pair_coeff

timestep        0.002  #ps

thermo_style    custom step temp pe ke etotal press vol lx ly lz xy xz yz
thermo          ${THERMO_FREQ}

# DUMP
dump            dpgen_dump all custom 10 traj/*.lammpstrj id type x y z

# equil - NPT_ISO
velocity        all create ${TEMP1} ${SEED}

fix             1 all nvt temp ${TEMP1} ${TEMP1} ${TAU_T}
run             ${NSTEPS1}
unfix           1

```
**running commands:**
**train:** 
```
export DP_INTERFACE_PREC=low
dp train input.json
```
**md:** 
`lmp_low -i input.lammps`
**error log:**
```
 >>> Info of model(s):
  using   4 model(s): ../graph.002.pb ../graph.000.pb ../graph.001.pb ../graph.003.pb 
  rcut in model:      8
  ntypes in model:    1
Neighbor list info ...
  update every 1 steps, delay 10 steps, check yes
  max neighbors/atom: 2000, page size: 100000
  master list distance cutoff = 9
  ghost atom cutoff = 9
  binsize = 4.5, bins = 5 3 3
  1 neighbor lists, perpetual/occasional/extra = 1 0 0
  (1) pair deepmd, perpetual
      attributes: , newton on
      pair build: full/bin/atomonly
      stencil: full/bin/3d
      bin: standard
Setting up Verlet run ...
  Unit style    : metal
  Current step  : 0
  Time step     : 0.002
2021-08-30 18:44:26.033883: I tensorflow/stream_executor/platform/default/dso_loader.cc:53] Successfully opened dynamic library libcublas.so.11
2021-08-30 18:44:26.386775: I tensorflow/stream_executor/platform/default/dso_loader.cc:53] Successfully opened dynamic library libcublasLt.so.11
Segmentation fault (core dumped)
```

**Comment by njzjz** [comment]
Reproduced.

**Comment by njzjz** [comment]
Fixed by #1069 1069.
